fe/fe-core/src/main/java/com/starrocks/sql/analyzer/QueryAnalyzer.java

```diff
@@ -1153,7 +1153,11 @@ private Table resolveTable(TableRelation tableRelation) {
                     }
                 }
             } else {
-                table = metadataMgr.getTable(catalogName, dbName, tbName);
+                // treat it as a temporary table first
+                table = metadataMgr.getTemporaryTable(session.getSessionId(), catalogName, db.getId(), tbName);
```

Comment:
just call MetaUtils.getTableSessionAware()?

Reply:
> just call MetaUtils.getTableSessionAware()?

MetaUtils.getSessionAwareTable will getDatabase first. We already have Database and matadataMgr here, better to invoke metadataMgr's interface to avoid unnecessary overhead.